CodingPractice/PythonAssignments/cleancode/CountItemsInList.py

```diff
@@ -0,0 +1,6 @@
+def count_items_in_list(l):
+    abc = 0
+    for e in l:
+        abc = 0
```

Comment:
I can see the problem here, Basil. It appears you are changing the count variable inside of the for loop.